tests/integration/docusaurus/rule_based_profiler/multi_batch_example.py

```diff
@@ -0,0 +1,141 @@
+        max_value: $parameter.row_count_range.value.max_value
+        mostly: $variables.mostly
+        meta:
+          profiler_details: $parameter.row_count_range.details
+  column_ranges_rule:
+    domain_builder:
+      class_name: SimpleSemanticTypeColumnDomainBuilder
+      semantic_types:
+        - numeric
+      # BatchRequest yielding exactly one batch (March, 2019 trip data)
+      batch_request:
+        datasource_name: taxi_pandas
+        data_connector_name: monthly
+        data_asset_name: my_reports
+        data_connector_query:
+          index: -1
+    parameter_builders:
+      - parameter_name: min_range
+        class_name: NumericMetricRangeMultiBatchParameterBuilder
+        batch_request:
+            datasource_name: taxi_pandas
+            data_connector_name: monthly
+            data_asset_name: my_reports
+            data_connector_query:
+              index: "-6:-1"
+        metric_name: column.min
+        metric_domain_kwargs: $domain.domain_kwargs
+        false_positive_rate: $variables.false_positive_rate
+        round_decimals: 2
+      - parameter_name: max_range
+        class_name: NumericMetricRangeMultiBatchParameterBuilder
+        batch_request:
+            datasource_name: taxi_pandas
+            data_connector_name: monthly
+            data_asset_name: my_reports
+            data_connector_query:
+              index: "-6:-1"
+        metric_name: column.max
+        metric_domain_kwargs: $domain.domain_kwargs
+        false_positive_rate: $variables.false_positive_rate
+        round_decimals: 2
+    expectation_configuration_builders:
+      - expectation_type: expect_column_min_to_be_between
+        class_name: DefaultExpectationConfigurationBuilder
+        module_name: great_expectations.rule_based_profiler.expectation_configuration_builder
+        column: $domain.domain_kwargs.column
+        min_value: $parameter.min_range.value.min_value
+        max_value: $parameter.min_range.value.max_value
+        mostly: $variables.mostly
+        meta:
+          profiler_details: $parameter.min_range.details
+      - expectation_type: expect_column_max_to_be_between
+        class_name: DefaultExpectationConfigurationBuilder
+        module_name: great_expectations.rule_based_profiler.expectation_configuration_builder
+        column: $domain.domain_kwargs.column
+        min_value: $parameter.max_range.value.min_value
+        max_value: $parameter.max_range.value.max_value
+        mostly: $variables.mostly
+        meta:
+          profiler_details: $parameter.max_range.details
+"""
+
+data_context = DataContext()
+
+
+# Instantiate Profiler
+full_profiler_config_dict: dict = yaml.load(profiler_config)
+
+profiler: Profiler = Profiler(
+    profiler_config=full_profiler_config_dict,
+    data_context=data_context,
+)
+
+suite = profiler.profile()
+print(suite)
+
+# Please note that this docstring is here to demonstrate output for docs.
+first_rule_suite = """
```

Comment:
Perhaps a nitpick:
If you make this a proper dictionary (perhaps `expected_suite = {...}` rather than a string you could get these benefits:
1. You could write a single assertion at the end of the script `assert suite == expected_suite` increasing the robustness of the tutorial and ensuring no drift in this tutorial.
2. It could still be shown to the user and also have syntax highlighting, which the user won't because they will get a `print()` output, but that's still nice to show.
3. Less risk of breakage should the generated suite ever change.